apps/web/src/stores/documentLogWithMetadata.ts

```diff
@@ -11,7 +12,7 @@ export default function useDocumentLogWithMetadata(
       if (!documentLogUuid) return []
 
       const response = await fetch(
-        `/api/documentLogs/uuids/${documentLogUuid}`,
+        ROUTES.api.documentLogs.uuids.detail({ uuid: documentLogUuid }).root,
```

Comment:
Remove `.uuids.` doesn't add value and is confusing

Reply:
yeah i would actually wouldn't have used detail  which is the really confusing part

Reply:
detail is what we use in the other routes

Reply:
yeah but this is rest paths in which a detail noun makes no really sense, but anyhow we are really just discussing minor stuff here